{"frames":[{"func":"print(df_article_fact_part_2)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in print(df_article_fact_part_2) : \n  object 'df_article_fact_part_2' not found\n"}